pom.xml

```diff
@@ -57,7 +57,7 @@
         <checkstyle.suppressions.location>checkstyle/suppressions.xml</checkstyle.suppressions.location>
         <io.confluent.rest-utils.version>8.0.0-0</io.confluent.rest-utils.version>
         <conscrypt.version>2.5.2</conscrypt.version>
-        <jetty.servlets.version>9.4.53.v20231009</jetty.servlets.version>
+        <jetty.override.version>12.0.16</jetty.override.version>
```

Comment:
manual override for now to allow for local building. This is until the confluentinc/common is bumped to jetty 12, so we should remove this from the PR before we merge